src/main/java/seedu/address/ui/PersonListPanel.java

```diff
@@ -35,9 +34,8 @@
     /**
      * Creates a {@code PersonListPanel} with the given {@code ObservableList}.
      */
-    public PersonListPanel(ObservableList<Person> personList, Summary summary) {
+    public PersonListPanel(ObservableList<Person> personList, Summary summary, CommandBox commandBox) {
```

Comment:
I see. So you are tracking the CommandBox as a workaround

Reply:
More accurately, I am passing the singular CommandBox instance to PersonListPanel to update the view command upon a mouse click event in the left pane.